lang/src/main/antlr4/org/kaazing/k3po/lang/parser/v2/Robot.g4

```diff
@@ -32,7 +32,7 @@ streamNode
     ;
 
 acceptNode
-    : k=AcceptKeyword acceptURI=location ( AsKeyword text=Name )?
+    : k=AcceptKeyword (OptionKeyword TransportKeyword transport=TextLiteral AcceptKeyword)? acceptURI=location ( AsKeyword text=Name )?
```

Comment:
should we use transport=TextLiteral or some URI. At present, the http test is string (but the contents are URI)